lib/foreman_theme_satellite/branded_words.rb

```diff
@@ -7,50 +7,17 @@ module ForemanThemeSatellite
   # For now, modifiers are not supported by the generator.
   # Make sure the regexp works both with Ruby and JavaScript.
   # Entries are processed in order and first wins.
+  # Occurrences of "proxy" should be treated as a bug as it is unclear whether the author refers to a HTTP(S) proxy or a Capsule
   FOREMAN_BRAND = {
-    /%{proxy}/               => '%{proxy}',
-    /%{foreman}/             => '%{foreman}',
-    /\b[Pp]roxy [Hh][Tt][Tt][Pp]\b(?!-)/    => 'Proxy HTTP', # Workaround for French translation
-    /\b[Pp]roxies [Hh][Tt][Tt][Pp]\b(?!-)/ => 'Proxies HTTP', # Workaround for French translation
-    /\b[Hh][Tt][Tt][Pp] [Pp]roxy\b(?!-)/    => 'HTTP Proxy',
-    /\b[Hh][Tt][Tt][Pp] [Pp]roxies\b(?!-)/ => 'HTTP Proxies',
-    /\bHTTP\(S\) proxy\b(?!-)/ => 'HTTP(S) proxy',
-    /\bIgnore Proxy\b(?!-)/  => 'Ignore Proxy',
-    /\bIgnore proxy\b(?!-)/  => 'Ignore Proxy',
-    /\bSystem-wide proxies\b(?!-)/  => 'System-wide proxies',
-    /\bLeave this blank if no proxy is used\b(?!-)/ => 'Leave this blank if no proxy is used',
-    /\bbypass proxy settings\b(?!-)/ => 'bypass proxy settings',
-    /\bURL of the proxy\b(?!-)/ => 'URL of the proxy',
-    %r{\bhttps://proxy.example.com:8080\b(?!-)} => 'https://proxy.example.com:8080',
-    /\bproxy for all outgoing HTTP connections\b(?!-)/ => 'proxy for all outgoing HTTP connections',
-    /\bperform various actions through those proxies\b(?!-)/ => 'perform various actions through those proxies',
```

Comment:
Those strings are used as workarounds to blacklist the Proxy -> Capsule rule. In those strings the meaning of proxy is HTTP proxy, so they shouldn't be branded as Capsule. I am for removing those, but only after the strings are dropped from the code. Otherwise we will have those strings branded to Capsules, which is not desirable.

Reply:
That's fair, I have re-added those and added a comment.